README.md

```diff
@@ -126,7 +126,7 @@ running `docker-credential-ecr-login` will output: `command not found`
 You can install this via `go get` with:
```

Comment:
Would you mind updating this line too?

Reply:
Sure thing, I've done so in cdea1b160da480e1c6917854ff53acb06067de73